src/lib/viewers/controls/color-picker/ColorPickerToggle.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import ColorPickerPalette from './ColorPickerPalette';
+import { AnnotationMode } from '../annotations/types';
+import './ColorPickerToggle.scss';
+
+export type Props = {
+    isActive?: boolean;
+    annotationMode?: AnnotationMode;
+};
+
+export default function ColorPickerToggle({ annotationMode, isActive = false, ...rest }: Props): JSX.Element | null {
+    const [isColorPickerToggled, setIsColorPickerToggled] = useState(false);
+
+    if (annotationMode !== AnnotationMode.DRAWING) {
+        return null;
+    }
+
+    return (
+        <div className="bp-ColorPickerToggle">
+            {isColorPickerToggled && (
+                <ColorPickerPalette
+                    onColorSelect={(): void => {
```

Comment:
Later when I implement the business logic for updating the selected color, the parameter will no longer be empty, we will be passing in `color: string` instead.